### PR TITLE
update pseudocode

### PR DESCRIPTION
**Pull request by ymatsumu**
Updated pseudocode based on comment received.

### CODE REVIEW

File: codeFile/__pycache__/__init__.cpython-39.pyc
Comment (andrewfullard) [inline codeFile/__pycache__/__init__.cpython-39.pyc:1]: Don't commit python cache files

File: codeFile/spectra.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+# from scipy.stats import norm
+from scipy.optimize import curve_fit 
+# from scipy import optimize
+
+
+class Spectra:
+    def __init__(self, filePath):
+        self.filePath = filePath
+
+    def readData(self, cutOffLeft=6020, cutOffRight=6420):
```
Comment (andrewfullard) [inline codeFile/spectra.py:16]: docstrings

File: pseudocode.txt [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# standardize the data format
+def changeFileExtensions(folderPath, newFileType):
```
Comment (andrewfullard) [inline pseudocode.txt:2]: Changing file extensions does not change the type of the file. You need to rewrite them if you want to e.g. transform tab-separated to comma-separated

Comment (ymatsumu) [reply pseudocode.txt:2]: Oh, so this code does not work for other files! The code works to open the .flm files on the CfA Supernova Data Archive, so I will mention that in the documentation. I will write code to deal with other types of data later if time allows!

**Pull request merged**